test/server/cards/02_SHD/leaders/HondoOhnakaThatsGoodBusiness.spec.ts

```diff
@@ -0,0 +1,136 @@
+            });
+        });
+
+        describe('Hondo Ohnaka\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        resources: ['privateer-crew', 'warbird-stowaway', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst'],
+                        groundArena: ['battlefield-marine'],
+                        leader: 'hondo-ohnaka#thats-good-business',
+                    },
+                    player2: {
+                        spaceArena: ['green-squadron-awing'],
+                        resources: ['freetown-backup', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst']
+                    },
+                });
+            });
+
+            it('should give experience token when play a unit from smuggle', function () {
+                const { context } = contextRef;
+
+                // play a unit from smuggle
+                context.player1.clickCard(context.privateerCrew);
+
+                // choose between 2 triggers
+                context.player1.clickPrompt('Exhaust this leader');
+                expect(context.player1).toHavePassAbilityPrompt('Exhaust this leader');
+                context.player1.clickPrompt('Exhaust this leader');
+
+                // add experience to battlefield marine
+                expect(context.player1).toBeAbleToSelectExactly([context.battlefieldMarine, context.privateerCrew, context.greenSquadronAwing]);
+                context.player1.clickCard(context.battlefieldMarine);
+                expect(context.player2).toBeActivePlayer();
+                expect(context.battlefieldMarine).toHaveExactUpgradeNames(['experience']);
+            });
+
+            it('should give experience token when play a unit from smuggle (pass on first smuggle played)', function () {
+                const { context } = contextRef;
+
+                // play a unit from smuggle
+                context.player1.clickCard(context.privateerCrew);
+
+                // choose between 2 triggers
+                context.player1.clickPrompt('Exhaust this leader');
+                expect(context.player1).toHavePassAbilityPrompt('Exhaust this leader');
+
+                // do not use hondo ability yet
+                context.player1.clickPrompt('Pass');
+                expect(context.player2).toBeActivePlayer();
+
+                // opponent play a unit from smuggle, nothing happen
+                context.player2.clickCard(context.freetownBackup);
+                expect(context.player1).toBeActivePlayer();
+
+                // play a second unit from smuggle
+                context.player1.clickCard(context.warbirdStowaway);
+                expect(context.player1).toHavePassAbilityPrompt('Exhaust this leader');
+                context.player1.clickPrompt('Exhaust this leader');
```

Comment:
Somewhere we need at least one test that Hondo becomes exhausted and that the ability can't activate if he is